framework/auth/cas.py

```diff
@@ -160,17 +160,18 @@ def _parse_profile(self, raw, access_token):
 
     def revoke_application_tokens(self, client_id, client_secret):
         """Revoke all tokens associated with a given CAS client_id"""
-        url = self.get_application_revocation_url()
-        data = {'client_id': client_id,
-                'client_secret': client_secret}
+        return self.revoke_tokens(payload={'client_id': client_id, 'client_secret': client_secret})
 
-        resp = requests.post(url, data=data)
+    def revoke_tokens(self, payload):
```

Comment:
Since this is a helper method internal to the workings of this class, we should prefix the name with an underscore to indicate that other developers using this class should not call this method directly. 

`_revoke_tokens`